api/bibliotheca.py

```diff
@@ -107,8 +107,6 @@ class BibliothecaAPI(BaseCirculationAPI, HasSelfTests):
     AUTHORIZATION_HEADER = "3mcl-Authorization"
     VERSION_HEADER = "3mcl-Version"
 
-    MAX_METADATA_AGE = timedelta(days=180)
```

Comment:
We no longer cache 'metadata' representations, since we also now use them to get circulation information.